common/auto-updater/versions-db-manager.cpp

```diff
@@ -0,0 +1,307 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+#include <fstream>
+#include <unordered_set>
```

Comment:
Seem not to be used - can it be removed?

Reply:
Agreed

Reply:
Done